docs/version-history/lts-versions-10-0.rst

```diff
@@ -44,6 +44,10 @@ Bugs Fixed:
   configuration could be active at the same time.
   :jira:`1590`
 
+- Improved the HTCondor's systemd configuration to not start HTCondor
+  until remote filesystems (if any) have been mounted.
```

Comment:
@timtheisen this point is subtle, but maybe important. I belief that `After=remote-fs.target` will start HTCondor after Systemd _attempts_ to mount remote filesystems. It allows those attempts to fail so that HTCondor will still start even if there are improper configurations or remote filesystems are down.

(A) Do you agree with that being the behavior of After=remote-fs.target?
(B) I believe this behavior is desirable, do you?
(C) I would change "have been mounted" to something a bit more like "have been mounted. If mounting failures occur, HTCondor will still start."